<?xml version="1.0" ?>
<launch>    
    
<!-- Configuration file parameters -->
<param name="offset_file_path" value="$(find orbo_manager)/config/offset_orbo.yaml"/>
<param name="robot_file_path"  value="$(find orbo_manager)/config/orbo.robot"/>
<param name="init_file_path"   value="$(find orbo_manager)/config/dxl_init_orbo.yaml"/>
    
<!-- LEGS Manager -->
<node pkg="orbo_manager" type="orbo_manager" name="orbo_manager" output="screen">
</node>

</launch>
